File: integration-test/src/test/java/com/promise/integrationtest/server/message/ServerServerGroupMessage.java
package com.promise.integrationtest.server.message;

public enum ServerServerGroupMessage
{
    DeleteDefault ("Server.Message.ServerServerGroupDeleteDefault");
    private String id;

    ServerServerGroupMessage(String id)
    {
        this.id = id;
    }

    public String getId()
    {
        return id;
    }
}

File: server/object/message/ServerServerGroupMessage.go
package message

import (
	"promise/base"
)

const (
	// MessageServerServerGroupDeleteDefault Message ID
	MessageServerServerGroupDeleteDefault = "ServerServerGroup.Message.DeleteDefault"
)

// NewMessageServerServerGroupDeleteDefault will return a message.
func NewMessageServerServerGroupDeleteDefault() *base.Message {
	ret := base.NewMessage()
	ret.ID = MessageServerServerGroupDeleteDefault
	ret.Severity = base.SeverityWarning
	ret.Description = "Delete default server-servergroup is not allowed."
	return ret
}
